src/metemcyber/cli/cli.py

```diff
@@ -2169,14 +2170,33 @@ def _load_misp_event(filepath):
 @misp_app.command("event", help="Show exported MISP events")
 def misp_event(ctx: typer.Context):
     json_dumpdir = Path(_load_config(ctx)['misp']['download'])
+    # load metadata pickle for efficiency
+    metadata_cache_path = Path(_load_config(ctx)['misp']['download'], '../metadata_cache.pkl')
+    if os.path.exists(metadata_cache_path):
```

Comment:
isfileを使って確認した方がよさそう
```suggestion
    if os.path.isfile(metadata_cache_path):
```